src/app/components/layout/header/top-bar/top-bar.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="buttons-left">
   <span class="last-updated-time" *ngIf="isBalanceObtained">
-    {{ 'header.updated1' | translate }} {{ timeSinceLastUpdateBalances === 0 ? ('top-bar.less-than' | translate ) : timeSinceLastUpdateBalances }} {{ 'header.updated2' | translate }}
+    {{ 'header.top-bar.updated1' | translate }} {{ timeSinceLastUpdateBalances === 0 ? ('header.top-bar.less-than' | translate ) : timeSinceLastUpdateBalances }} {{ 'header.top-bar.updated2' | translate }}
     <i *ngIf="!this.isBalanceUpdated"
        class="material-icons blinking-warning-icon"
        [matTooltip]="'top-bar.connection-error-tooltip' | translate">
```

Comment:
`top-bar.connection-error-tooltip` should be `header.top-bar.connection-error-tooltip`